lib/app/server.js

```diff
@@ -153,15 +153,14 @@ export default async ssrContext => {
   ** Call .validate()
   */
   let isValid = true
-  let validPromises = await Promise.all(Components.map((Component) => {
+  await Promise.all(Components.map((Component) => {
     if (!isValid) return
     if (typeof Component.options.validate !== 'function') return
     let promise = promisify(Component.options.validate, app.context)
     promise.then(result => {
       isValid = result;
       return result;
     })
-    return Promise.all([promise])
```

Comment:
I mean no need to use `Promise.all` 😄 , but `return promise` still necessary.

Or you can just `return promisify(Component.options.validate, app.context).then(...)`